vonage/assets/logs/vonage_tests.yaml

```diff
@@ -0,0 +1,94 @@
+id: "vonage"
+tests:
+ -
+  sample: |-
+    {
+      "country" : "IN",
```

Comment:
I tested this integration and it seems fine, however could we have `error_code_description`, `product`, `error_code` and `latency` as a tag (under `ddtags`) so it's testing the remappers